libs/BehaviorKit/source/BehaviorKit.cpp

```diff
@@ -84,6 +84,9 @@ void BehaviorKit::bleConnection(bool with_video)
 	_ledkit.start(&led::animation::ble_connection);
 	if (with_video) {
 		_videokit.playVideoOnce("/fs/home/vid/system/robot-system-ble_connection-wink-no_eyebrows.avi");
+	} else {
+		rtos::ThisThread::sleep_for(5s);
+		blinkOnCharge();
```

Comment:
This function (bleConnection) does not have notion of charge, only of video.

This workaround is fine but bring some confusion in BehaviorKit implementation.